tasks/certificate.yml

```diff
@@ -13,3 +13,14 @@
     {% endif %}
     {% if letsencrypt_staging %} --staging {% endif %}"
   when: not letsencrypt_cert.stat.exists
+
+- name: Force generation of a new certificate
+  shell: >
+    "certbot certonly --force-renewal --nginx --email '{{ letsencrypt_email }}'
+    --agree-tos -d '{{ domain_name }}'
+    {% if certbot_nginx_cert_name is defined %}
+    --cert-name '{{ certbot_nginx_cert_name }}'
+    {% endif %}
+    {% if letsencrypt_staging %} --staging {% endif %}"
+  when: letsencrypt_cert.stat.exists and certbot_force_update is defined
+  notify: reload ngninx
```

Comment:
```suggestion
  notify: reload nginx
```

by @delphaber

Reply:
@Matt-Yorkley , when you fix the nginx typo I'll test this and merge it if everything ok. It's been too long for this feature, sorry I was out.

Reply:
Fixed. It's not really urgent, but we do have some certificates on a couple of servers that need to be updated.